include/rocksdb/statistics.h

```diff
@@ -320,6 +320,8 @@ enum Tickers : uint32_t {
   // # of times snapshot_mutex_ is acquired in the fast path.
   TXN_SNAPSHOT_MUTEX_OVERHEAD,
 
+  // Number of keys actually found in MultiGet calls (vs number requested by caller)
```

Comment:
which one is number requested by caller? `NUMBER_MULTIGET_KEYS_READ`? maybe clarify